src/cascadia/LocalTests_TerminalApp/SettingsTests.cpp

```diff
@@ -56,8 +56,6 @@ namespace TerminalAppLocalTests
         TEST_METHOD(TestInvalidColorSchemeName);
         TEST_METHOD(TestHelperFunctions);
 
-        TEST_METHOD(TestLayerGlobalsOnRoot);

```

Comment:
You're right that we don't need a test for this anymore. But now I'm curious, what happens if a user still had globals? Is everything inside globals just ignored?

Reply:
Yes, it’s ignored.